src/test/external-git/gitea-values.yaml

```diff
@@ -0,0 +1,34 @@
+persistence:
+  storageClass: "standard" # 'standard' is for kind
```

Comment:
This might cause us problems right?  Because k3s/k3d don't have that storageclass ootb, which is why we have the storage class template variable.

Reply:
Good catch, I was defiantly only using kind when I was testing since that is what we have the simple `make dev-agent-image` target to replace the container image for the  Zarf agent.

I originally intended for this test to be manually run for now since there wasn't a clear place for it to exist within the the e2e test for every PR (I didn't want to have to slow down the feedback loop by doing a `zarf destroy` and re-initializing another cluster.

Are you okay with keeping it 'dumb' and having assumptions over the distro? Or should I add it to the e2e maybe as its own standalone workflow?